apps/backend/src/indicateurs/trajectoires/trajectoires-data.service.spec.ts

```diff
@@ -3,7 +3,7 @@ import { PermissionService } from '@/backend/users/authorizations/permission.ser
 import { Test } from '@nestjs/testing';
 import CollectivitesService from '../../collectivites/services/collectivites.service';
 import SheetService from '../../utils/google-sheets/sheet.service';
-import { ListDefinitionsService } from '../list-definitions/list-definitions.service';
+import { ListDefinitionsService } from '../definitions/list-definitions/list-definitions.service';
```

Comment:
Cet import n'est pas le bon je pense.

```suggestion
import { ListDefinitionsService } from '../definitions/list-definitions/list-definitions.service';
```